velox/dwio/parquet/writer/arrow/ThriftInternal.h

```diff
@@ -506,7 +510,9 @@ class ThriftDeserializer {
 /// to treat it as a string.
 class ThriftSerializer {
  public:
-  explicit ThriftSerializer(int initial_buffer_size = 1024)
+  explicit ThriftSerializer(
+      int initial_buffer_size =
+          FLAGS_velox_arrow_parquet_writer_thrift_serialized_buffer_size)
```

Comment:
GFlags are hard to manage. Would it be possible to use a config property instead?